Admin.API/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Admin.API.Configuration;
+using Microsoft.OpenApi.Models;
+
+namespace Admin.API.Extensions;
+
+public static class IServiceCollectionExtensions
+{
+    public static IServiceCollection AddSwaggerWithCustomUi(this IServiceCollection services,
```

Comment:
This was bugging me because it implies that we're adding a custom Swagger Ui, but the truth is that if you don't call this, Swagger is not generated/made available. I figured this new name made more sense, but I'll revert it.

Reply:
Okay, I get you. Never thought about it his way.

The reason for this naming is a naming convention I introduced. There are multiple methods like this, for example AddCustomIdentity, AddCustomAspNetCore, ...
The problem is that most of the time, there are methods like AddIdentity or AddAspNetCore. In order to avoid naming conflicts, I added the `Custom` prefix. 
Do you have a better idea for it? It would be great if the naming convention would be applicable to all cases.

Reply:
Out of the top of my head, I'd say `AddCustomSwaggerWithUi`.
I think the Custom keyword makes sense, so this is a way of keeping it.

Reply:
Okay, I aggree